Experiments/Experiments/DefaultFeatureFlagService.swift

```diff
@@ -48,7 +48,7 @@ public struct DefaultFeatureFlagService: FeatureFlagService {
         case .simplifiedLoginFlowI1:
             return buildConfig == .localDeveloper || buildConfig == .alpha
         case .justInTimeMessagesOnDashboard:
-            return buildConfig == .localDeveloper || buildConfig == .alpha
+            return true
```

Comment:
Was this meant to go in this PR?

Reply:
Yep, that's intentional, this is the last bit required for the release of i1. I've checked that it works correctly on a release build.

Should have called it out in the description, sorry!